addons/mail/models/fetchmail.py

```diff
@@ -182,7 +182,7 @@ def button_confirm_login(self):
     @api.model
     def _fetch_mails(self):
         """ Method called by cron to fetch mails from servers """
-        return self.search([('state', '=', 'done'), ('server_type', '!=', 'local')]).fetch_mail()
+        return self.search([('state', '=', 'done'), ('server_type', '!=', 'local')]).with_context(call_by_cron=True).fetch_mail()
```

Comment:
it's not a big deal imo, can we do it in master instead with a new argument to `fetch_mail` which will be `raise_exception` set at `False` by default ?

A bit like the method `_send` on `mail.mail`

What do you think ?

Also the commit message can be improved a bit (we generally try to not exceed 72 chars by line, do not give your opinion like "I agree" by only give facts, etc)

Reply:
Ok , i get it, let close this PR and move to master later. Thank for your response